README.md

```diff
@@ -120,6 +119,28 @@ Here's an example of what you should expect:
 > - You can run the script as many times as you want. It will only index the pages that are not already indexed.
 > - Sites with a large number of pages might take a while to index, be patient.
 
+## API
```

Comment:
I think people might get confused and think this is a Web API, maybe we replace it with one of these?
```
## Programmatic Usage
## Using as NPM Module
```

?

Reply:
Totally agree, I'll sort out the conflicts and change that!